src/controllers/BoardController.js

```diff
@@ -48,6 +49,14 @@ const createBoard = async ( req, res ) => {
 
     const board = await BoardService.createBoard( req.body );
 
+    const doneColumn = {
```

Comment:
Maybe also add a ToDo column or Backlog column and since this is service code move this to the service layer.